Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.py

```diff
@@ -1609,12 +1619,15 @@ def revoke_security_group_egress_command(args, aws_client):
     kwargs = {
         'GroupId': args.get('groupId')
     }
+    if args.get('IpPermissionsFull', None):
+        IpPermissions = json.loads(args.get('IpPermissionsFull'))
+        kwargs['IpPermissions'] = IpPermissions
+    else:
+        IpPermissions_dict = create_ip_permissions_dict(args)
+        UserIdGroupPairs_dict = create_user_id_group_pairs_dict(args)
 
-    IpPermissions_dict = create_ip_permissions_dict(args)
-    UserIdGroupPairs_dict = create_user_id_group_pairs_dict(args)
-
-    IpPermissions_dict['UserIdGroupPairs'] = [UserIdGroupPairs_dict]
-    kwargs['IpPermissions'] = [IpPermissions_dict]
+        IpPermissions_dict['UserIdGroupPairs'] = [UserIdGroupPairs_dict]
+        kwargs['IpPermissions'] = [IpPermissions_dict]
```

Comment:
Why here IpPermissions_dict is being appended to a list and in line 1624 not?

Reply:
The assumption is that you would be passing it a list object already.  Example:
```
all_traffic_rule = """[{"IpProtocol": "-1", "IpRanges": [{"CidrIp": "0.0.0.0/0"}], "Ipv6Ranges": [], "PrefixListIds": [], "UserIdGroupPairs": []}]"""
return_results(demisto.executeCommand("aws-ec2-revoke-security-group-ingress-rule", {"groupId":'sg-0aa2f27bb524c2e01', "IpPermissionsFull":all_traffic_rule}))
```

Reply:
So maybe it should be documented in the argument description and README.

Reply:
made changes in [add ippermissionsfull example](https://github.com/demisto/content/pull/25209/commits/df1a589027cd09dcda6a145316415c5e645fd46c)